src/Widgets/LeeanPreview.php

```diff
@@ -0,0 +1,167 @@
+<?php namespace Leean\Widgets\Widgets;
```

Comment:
The first Widgets is for the package. The second is for the set of pre-defined Widgets supplied with it. Can you think of a better naming convention?